geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -10219,6 +10219,7 @@ public void initializeStats(long numEntriesInVM, long numOverflowOnDisk,
       long numOverflowBytesOnDisk) {
     getDiskRegion().getStats().incNumEntriesInVM(numEntriesInVM);
     getDiskRegion().getStats().incNumOverflowOnDisk(numOverflowOnDisk);
+    getDiskRegion().getStats().incNumOverflowBytesOnDisk(numOverflowBytesOnDisk);
```

Comment:
I would put this change in a different PR/JIRA as it is about a different bug.

Reply:
Thanks for the comment. I'll definitely move this change to the correct JIRA.